src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDaoImpl storageDao;
+    private static User newUser;
+    private static User actual;
+    private static final int MIN_AGE = 18;
+    private static final String CORRECT_LOGIN = "loginLog";
+    private static final String CORRECT_PASSWORD = "password";
+
+    @BeforeAll
+    static void setUp() {
+        newUser = new User();
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setNewUser() {
+        newUser.setAge(MIN_AGE);
+        newUser.setLogin(CORRECT_LOGIN);
+        newUser.setPassword(CORRECT_PASSWORD);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void noSuchUsersLoginInStorage_Ok() {
+        newUser.setLogin("abfhlfjsn");
+        actual = storageDao.get(newUser.getLogin());
+        assertNotEquals(newUser, actual);
+    }
```

Comment:
You should test registration service, not dao.
Positive case will just obtain new user for registration
Negative case should register user and then obtain second call with user with same login